src/components/Script.jsx

```diff
@@ -24,9 +24,9 @@ const Script = ({ actors, scriptOption }) => {
   let currentActorCharacter;
 
   for (let i = 0; i < script.length; i++) {
-    // assign the character name as the className for the div
-    const name = script[i][0].replace('.', '').toLowerCase();
-    if (characterSet.has(name.toUpperCase())) {
+    // checks if the character name for this chunk is the name of a character assigned to the current actor
+    const name = script[i][0].replace('.', '').toUpperCase();
```

Comment:
Please add a comment noting the assumption that play scripts always have character names in all caps. We may find plays without that convention at some point.

Reply:
Also on why `.` are being striped out

Reply:
The new way of grabbing the character data doesn't require the to uppercase. The dot is right after each character name though, I was removing it for there to be a more accurate comparison. I've added a comment.